### DISCUSSION TITLE
在Netlify上部署

### DISCUSSION THREAD

**Discussion by ghost** [Qexo/Qexo#363] [Q&A] [answered]
我想要在Netlify上部署qexo，按照文档配置后提示部署成功，但打开后却是404

**Comment by Fgaoxing** [comment]
不支持netlify

**Comment by am-abudu** [comment]
我印象里没有写过Netlify部署的文档。请告诉我你在哪里看到的

**Reply by ghost** [reply]
您好，感谢您的回复，这是我之前看到的博客链接：https://blog.kwxos.top/posts/18807.html

**Reply by am-abudu** [reply, marked as answer]
这个链接中描述的Netlify并不是部署的Qexo项目，请参照Qexo文档

**Comment by Fgaoxing** [comment]
我是没搜到，可能是像个人博客这种深网（不是暗网，深网通常指无法被搜索引擎搜索到的网站和页面）里的



在 2023-04-20 21:08:22，"Abudu" ***@***.***> 写道：

我印象里没有写过Netlify部署的文档。请告诉我你在哪里看到的

**Comment by Fgaoxing** [comment]
我一猜就是，我昨天也看到了，但是，这个他也不是Netlify呀，这个让你看原文档



在 2023-04-21 07:40:31，"chulan" ***@***.***> 写道：

您好，感谢您的回复，这是我之前看到的博客链接：https://blog.kwxos.top/posts/18807.html
